contrib_bots/lib/john.py

```diff
@@ -0,0 +1,125 @@
+# pip install chatterbot is necessary
+import os
+import json
+import time
+import threading
+from random import choice
+from chatterbot import ChatBot
+from chatterbot.trainers import ChatterBotCorpusTrainer, ListTrainer
+jokes_path = os.path.expanduser('lib/John/jokes.json')
+
+# Create a new instance of a ChatBot
+bot = ChatBot("John",
+              storage_adapter="chatterbot.storage.JsonFileStorageAdapter",
+              logic_adapters=
+              [
+                  "chatterbot.logic.MathematicalEvaluation",
+                  {
+                      "import_path": "chatterbot.logic.BestMatch",
+                      "response_selection_method": "chatterbot.response_selection.get_random_response",
+                      "statement_comparison_function": "chatterbot.comparisons.levenshtein_distance"
+                  }],
+              output_adapter="chatterbot.output.OutputFormatAdapter",
+              output_format='text',
+              database="lib/John/database.db",
+              silence_performance_warning="True")
+
+bot.set_trainer(ListTrainer)
+
+bot.train([
+    "I want to contribute",
+    """Contributors are more than welcomed! Please read
```

Comment:
Does this string show correctly? When you use triple quoted string the white-spaces are preserved. For example:
```Python
>>> f = """hello
...        world"""
>>> print(f)
hello
       world
>>>
```